src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+
+
+namespace System.Reflection.Emit.Experimental
+{
+    class AssemblyBuilder: System.Reflection.Assembly
+    {
+        internal AssemblyBuilder() { }
+
+        public void Save(string assemblyFileName)
+        {
+//New method
+        }
+
+        public static System.Reflection.Emit.AssemblyBuilder DefineDynamicAssembly(System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access) { throw new NotImplementedException("Under construction"); }// code forthecoming
+        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Defining a dynamic assembly requires dynamic code.")]
+
+        public static System.Reflection.Emit.AssemblyBuilder DefineDynamicAssembly(System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access, System.Collections.Generic.IEnumerable<System.Reflection.Emit.CustomAttributeBuilder>? assemblyAttributes) { throw new NotImplementedException("Under construction"); }
+        public System.Reflection.Emit.ModuleBuilder DefineDynamicModule(string name) { throw new NotImplementedException("Under construction"); }
```

Comment:
You do need have provide a message for NotImplementedException. `new NotImplementedException()` will work find and it will come with default message that is about as good.

Also, our [coding style](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md) is to always have braces on a new line:
```
        public ModuleBuilder DefineDynamicModule(string name)
        {
            throw new NotImplementedException();
        }
```
Or you can use "lamda syntax" for one line methods.
```
        public ModuleBuilder DefineDynamicModule(string name)
            => throw new NotImplementedException();
```